security/zero_trust_workload_identity_manager/zero-trust-manager-release-notes.adoc

```diff
@@ -7,10 +7,69 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The {zero-trust-full} leverages Secure Production Identity Framework for Everyone (SPIFFE) and the SPIFFE Runtime Environment (SPIRE) to provide a comprehensive identity management solution for distributed systems.
+The {zero-trust-full} leverages Secure Production Identity Framework for Everyone (SPIFFE) and the SPIFFE Runtime Environment (SPIRE) to provide a comprehensive identity management solution for distributed systems. {zero-trust-full} supports SPIRE version 1.12.4 running as an operand.
 
 These release notes track the development of {zero-trust-full}.
 
+[id="zero-trust-manager-release-notes-2"]
+== {zero-trust-full} 0.2.0 (Technology Preview)
+
+Issued: 2025-09-08
+
+There following advisories are available for the {zero-trust-full}.
+
+* https://access.redhat.com/errata/RHBA-2025:15426[RHBA-2025:15426]
+* https://access.redhat.com/errata/RHBA-2025:15427[RHBA-2025:15427]
+* https://access.redhat.com/errata/RHBA-2025:15425[RHBA-2025:15425]
+* https://access.redhat.com/errata/RHBA-2025:15428[RHBA-2025:15428]
+
+This release of {zero-trust-full} is a Technology Preview.
+
+[id="zero-trust-manager-features-enhancements_{context}"]
```

Comment:
This ID should probably include the version so that any further New features and enhancements sections have unique IDs, something like:
```suggestion
[id="zero-trust-manager-0-2-0-features-enhancements_{context}"]
```